src/sentry/static/sentry/app/views/eventsV2/eventDetails/lineGraph.tsx

```diff
@@ -109,17 +111,27 @@ const handleClick = async function(
     ? 'last_seen'
     : null;
 
+  const endValue = getUtcDateString(value + intervalToMilliseconds(interval));
+  const startValue = getUtcDateString(value);
+  const queryWithTime = `${queryString} timestamp:>${startValue} timestamp:<=${endValue}`;
```

Comment:
Is it worth double checking to make sure that timestamp is not already in the query? Not sure where the original queryString comes from.

Reply:
Good point. This is something we want to check.

Reply:
I can parse the query and replace existing timestamp conditions.